packages/plugins/color-picker/admin/src/index.ts

```diff
@@ -64,9 +66,9 @@ export default {
       },
     });
   },
-  async registerTrads({ locales }) {
+  async registerTrads({ locales }: { locales: string[] }) {
     const importedTrads = await Promise.all(
-      locales.map((locale) => {
+      locales.map(async (locale) => {
```

Comment:
Do you really need to make the callback async, when you are only building an array of promises?